core/domain/email_manager_test.py

```diff
@@ -6151,6 +6157,17 @@ def test_assign_translation_reviewer_email_for_can_send_emails_is_false(
             self.TRANSLATION_REVIEWER_EMAIL)
         self.assertEqual(len(messages), 0)
 
+    def test_assign_submit_question_email_for_can_send_emails_is_false(
+        self
+    ) -> None:
+        with self.can_not_send_emails_ctx:
+            email_manager.send_email_to_new_contribution_submit_questions(
+                self.submit_question_id)
+
+        messages = self._get_sent_email_messages(
+            self.TRANSLATION_REVIEWER_EMAIL)
```

Comment:
```suggestion
            self.QUESTION_CONTRIBUTOR_EMAIL)
```
This needs to be the question contributor's email.

Reply:
Done